### ISSUE TITLE
[LSP] Typing the colon of a named parameter in a Razor file results in a second colon

### ISSUE THREAD

**Issue by Tragetaschen** [dotnet/roslyn#60756]
**Describe the bug:**
When you type a named parameter in a method call, the colon character between the parameter name and the value appears twice resulting in a syntax error.

**Version used:**
VS2022 17.2.0 Preview 2.1

**To reproduce:**
```razor
@code {
    void M()
    {
        S(/* <-- type `a:1` here*/);
    }

    void S(int a = 1)
    { }
}
```
At the marked position, type `a`, `:`, and `1`

**Expected behavior:**
The method call should read `S(a:1);`

**Actual behavior:**
The method call reads `S(a::1)` with a double colon.

**Additional context:**
In a `.cs` file, the double colon is avoided.

**Comment by dibarbet** [comment]
This is fixed in 17.3 Preview 1

**Comment by dibarbet** [comment]
Oops this is not fixed, I had the wrong option on.  Will be taking a look at this shortly.

**Comment by dibarbet** [comment]
Ok, looks like this is fixed in a non-public preview version.  Will leave this open until it is released
![completion_commit_char](https://user-images.githubusercontent.com/5749229/169928680-ea02bc96-e4d5-488c-acc6-7f1fbe2ec5a5.gif)

**Comment by dibarbet** [comment]
Fixed in 17.3 Preview 2
